test/drive.js

```diff
@@ -0,0 +1,22 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+
+describe('drive', function() {
+
+  it('is not a variadic function', function() {
+    eq(typeof R.drive, 'function');
+    eq(R.drive.length, 2);
+  });
+
+  it('is a curried function', function() {
+    var x = 2;
+    var g = R.drive(R.__, [R.add(3)]);
+    eq(g(x), 5);
```

Comment:
This tests that it works with the placeholder, but it doesn't check the other way.